demo/2-istio-services.yaml

```diff
@@ -0,0 +1,64 @@
+# Istio Ingress Example for Project Jackson
+#
+# Authors:
```

Comment:
Necessary? VCS like Git make this information redudant

Reply:
It's something I grew accustomed to doing for open source projects where files might be moved between repos and git history lost; handy to know who to complain to if something breaks haha